linux/step04_all_omero.sh

```diff
@@ -2,24 +2,43 @@
 
 set -e -u -x
 
-source settings.env
+OMEROVER=${OMEROVER:-omero}
+PY_ENV=${PY_ENV:-py27}
 
+source settings.env
 #start-install
+if [ "$PY_ENV" = "py27_scl" ]; then
+	#start-py27-scl
+	set +u
+	source /opt/rh/python27/enable
+	set -u
+	#end-py27-scl
+fi
+
+if [[ ! $PY_ENV = "py27_ius" ]]; then
+	#start-venv
+	virtualenv /home/omero/omeroenv
+	/home/omero/omeroenv/bin/pip install omego
+	#end-venv
+fi
 
-SERVER=http://downloads.openmicroscopy.org/latest/omero5/server-ice35.zip
-
-wget $SERVER
-unzip -q server-ice35.zip
+#start-install
+if [ $OMEROVER = omerodev ]; then
+	/home/omero/omeroenv/bin/omego download --branch OMERO-DEV-latest server
```

Comment:
I think this is one of those arguments that can't be simplified, since I made them synonyms for convenience (such as the case above), even though `branch` and `release` have different meanings.